changelog/4136.splinedoc.rst

```diff
@@ -0,0 +1 @@
+Improvement for spline option doc for resampling.
```

Comment:
```suggestion
Clarified spline option for `sunpy.map.Map.resampling`.
```
This file should also be renamed `4136.docfix.rst`

Reply:
Done

Reply:
Thanks, can you also delete the old file?

Reply:
Yes, deleted the same.